cdhweb/events/admin.py

```diff
@@ -16,12 +16,17 @@ def tag_list(self, obj):
         return u", ".join(o.name for o in obj.tags.all())
     tag_list.short_description = 'Tags'
 
+    def is_virtual(self, obj):
+        return "Yes" if obj.is_virtual else "No"
+    is_virtual.short_description = 'Virtual'
+
```

Comment:
`Event` itself doesn't have this field; it only has a `@property` that delegates to `Location.is_virtual`.

Reply:
I tried it and it just says "True" or "False", no checkbox - I think because it's delegating to `Location` and isn't a true Field.

Reply:
right - you need to add `is_virtual.boolean = True` after the property is defined on the model

Reply:
oh wait, you can't do admin flags on a property; might be simpler to just use it as a method instead of a property; if you're mainly using it in templates there's no difference.

Reply:
you can also customize the label for your method in admin with this syntax:  `address_report.short_description = "Address"`